ci/taos/plugins-good/pr-format-doxygen-tag.sh

```diff
@@ -149,6 +154,12 @@ function pr-format-doxygen-tag(){
                     doxygen_rules="@package @brief"
                     doxygen_rule_num=0
                     doxygen_rule_all=0
+
+                    # When check_result is skip, set value to success(default)
```

Comment:
Please update the statement. 


`# When a current file is a source code, change a default value of $check_result from 'skip' to 'success'.`